integration-test/473-landuse-tier.py

```diff
@@ -0,0 +1,38 @@
+# http://www.openstreetmap.org/way/167274589
+# area 300363008
+assert_has_feature(
+    16, 10818, 21900, 'landuse',
```

Comment:
Should this test be for zoom 4 (the first zoom which would include this feature)?